Assets/Scripts/Terrain/StreamingWorld.cs

```diff
@@ -1250,7 +1250,7 @@ public void UpdateTerrainNature(TerrainDesc terrainDesc)
             if (dfTerrain && dfBillboardBatch)
             {
                 // Get current climate and nature archive and terrain distance
-                int natureArchive = ClimateSwaps.GetNatureArchive(LocalPlayerGPS.ClimateSettings.NatureSet, dfUnity.WorldTime.Now.SeasonValue);
+                int natureArchive = ClimateSwaps.GetNatureArchive(dfTerrain.MapData.worldClimate, dfUnity.WorldTime.Now.SeasonValue);
```

Comment:
Good catch. :) But can you please use MapsFile.GetWorldClimateSettings(worldClimate) here instead of a custom method. Like so:

    DFLocation.ClimateSettings worldClimateSettings = MapsFile.GetWorldClimateSettings(dfTerrain.MapData.worldClimate);
    int natureArchive = ClimateSwaps.GetNatureArchive(worldClimateSettings.NatureSet, dfUnity.WorldTime.Now.SeasonValue);

GetWorldClimateSettings() returns a DFLocation.ClimateSettings package from which you can then get nature archive. This keeps the conversation of climate index to nature archive all in the one place so changes won't become out of sync. It also simplifies the PR a bit. Cheers!

Reply:
Hey, I am new to Github and for me that's it. I am done with it an never want to work with it ever again. I tried it for one hour now and couldn't figure out how to change this PR. I think I closed the branch already by accident but who knows what branches, commits, detached heads and all this other crap really are...

Maybe you can just copy those two lines into your original code by yourself and delete this PR?
I just reached my threshold of tolerance and time I want to waste on this.
I am so done with Github...